docs/core/diagnostics/trace-perfcollect-lttng.md

```diff
@@ -0,0 +1,201 @@
+
+   >```bash
+   > sudo bash
+   > cp ~/.nuget/packages/runtime.linux-x64.microsoft.netcore.app/2.1.0/tools/crossgen /usr/share/dotnet/shared/Microsoft.NETCore.App/2.1.0
+   >```
+
+Once you have done this, `perfcollect` will use crossgen to include framework symbols. The warning that `perfcollect` used to issue should go away. This only has to be one once per machine (until you update your runtime).
+
+### Alternative: Turn off use of precompiled code
+
+If you don't have the ability to update the .NET Runtime (to add `crossgen`), or if the above procedure did not work for some reason, there is another approach to getting framework symbols. You can tell the runtime to simply not use the precompiled framework code. The code will be Just-In-Time compiled and the `crossgen` tool is not needed.
+
+> [!NOTE]
+> Choosing this approach may increase the startup time for your application.
+
+To do this, you can add the following environment variable:
+
+```bash
+export COMPlus_ZapDisable=1
+```
+
+With this change you should get the symbols for all .NET code.
+
+## Getting Symbols For the Native Runtime
+
+Most of the time you are interested in your own code, which `perfcollect` resolves by default. Sometimes it is very useful to see what is going on inside the .NET DLLs (which is what the last section was about), but sometimes what is going on in the native runtime dlls (typically libcoreclr.so), is interesting.  `perfcollect` will resolve the symbols for these when it converts its data, but only if the symbols for these native DLLs are present (and are beside the library they are for).
+
+There is a global command called [dotnet-symbol](https://github.com/dotnet/symstore/blob/master/src/dotnet-symbol/README.md#symbol-downloader-dotnet-cli-extension) which does this. To use dotnet-symbol to get native runtime symbols:
+
+1. Install `dotnet-symbol`:
+
+    ```bash
+    dotnet tool install -g dotnet-symbol
+    ```
+
+2. Download the symbols. If your installed version of the .NET Core runtime is 2.1.0 the command to do this is
+
+    ```bash
+    mkdir mySymbols
+    dotnet symbol --symbols --output mySymbols  /usr/share/dotnet/shared/Microsoft.NETCore.App/2.1.0/lib*.so
+    ```
+
+3. Copy the symbols to the correct place
+
+    ```bash
+    sudo cp mySymbols/* /usr/share/dotnet/shared/Microsoft.NETCore.App/2.1.0
+    ```
+
+After this, you should get symbolic names for the native dlls when you run `perfcollect`.
+
+## Collecting in a Docker Container
+
+For more information on how to use `perfcollect` in container environments, refer to [Collect diagnostics in containers](./diagnostics-in-containers.md) documentation.
+
+## Viewing a Trace
```

Comment:
I would move this trace viewing information above the section about resolving framework symbols. Assuming the trace was collected succesfully this is the logical next step in the workflow.